src/main/java/com/twilio/type/RecordingRulesUpdate.java

```diff
@@ -0,0 +1,43 @@
+package com.twilio.type;
```

Comment:
There are many similarities to the `SubscribeRulesUpdate.java` class. Similar to the `SubscribeRule` and `RecordingRule` classes, we can simplify with a base class.

Reply:
Done